lessons/flow.md

```diff
@@ -28,6 +28,10 @@ based on analysis of a program's sourcecode. If a program passes a static type c
 to satisfy some set of type safety properties for all possible inputs.[[1](https://en.wikipedia.org/wiki/Type_system#Static_type_checking)]
 This process usually runs at compile time.
 
+The alternative to state type checking is dynamic type checking. Dynamic type checking is the process of verifying the 
```

Comment:
"static"
- also is it worth saying there is a genuine alternative of "no type checking" or is that a given?